src/main/java/com/purbon/kafka/topology/model/Topic.java

```diff
@@ -1,24 +1,37 @@
 package com.purbon.kafka.topology.model;
 
-import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 import com.purbon.kafka.topology.TopologyBuilderConfig;
-import com.purbon.kafka.topology.model.Impl.TopicImpl;
 import com.purbon.kafka.topology.model.users.Consumer;
 import com.purbon.kafka.topology.model.users.Producer;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
 
-@JsonDeserialize(as = TopicImpl.class)
+/**
+ * Kafka topic representation.
+ *
+ * YAML:
+ * <pre>
+ * {@code
+ *     topics:
+ *       - name: "topicE"
+ *         config:
+ *           replication.factor: "1"
+ *           num.partitions: "1"
+ *       - name: "topicF"
+ *         config:
+ *           replication.factor: "1"
+ *           num.partitions: "1"
+ * }
+ * </pre>
+ */
 public interface Topic {
 
   String getName();
 
   List<TopicSchemas> getSchemas();
 
-  String getPlan();
```

Comment:
we need plans, this is a supporter future.

Reply:
Cool! Could you share some backing comments here to add it back?

Reply:
@purbon I think there are a need for some more tests in this area, testing Topic and Plan together -  right now I can only find PlanMapSerdesTest. 

It is a bit confusing to find an usused method on an interface for the model.